### DISCUSSION TITLE
Achievements?

### DISCUSSION THREAD

**Discussion by napoleonicroads** [Heroic-Games-Launcher/HeroicGamesLauncher#552] [Q&A]
Do you get achievements from the launcher and it goes to your epic account? Or if not are there any plans / or a chance to add them

**Comment by flavioislima** [comment]
We do not support achievements for now and we don't know if it will work. To be honest, I haven't tested if you get the achievements or not while using Heroic, we just don't show them for now. Maybe in the future but I have no idea on how they work so far since I don't use the official epic launcher for quite a while now.

**Comment by Arcitec** [comment]
Achievements are supported but requires manual work:

https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/1279#issuecomment-1939547292

**Comment by SuperSamus** [comment]
Technically, achievements on the Epic Games Store or supported out of the box with Legendary, but you get no notifications.
